apps/server/test/modules/task/task-finished.e2e-spec.ts

```diff
@@ -41,36 +41,36 @@ class API {
 }
 
 describe('Task controller (e2e)', () => {
-	describe('task/finished without permission', () => {
-		let app: INestApplication;
-		let em: EntityManager;
-		let currentUser: ICurrentUser;
-		let api: API;
-
-		beforeAll(async () => {
-			const module: TestingModule = await Test.createTestingModule({
-				imports: [ServerTestModule],
+	let app: INestApplication;
```

Comment:
The test now only work if they have the correct order. Can you please add by task/finished without permission'
That currentUser is set to undefined, to make it save?